Improve documentation (#1588)
swiftlang/vscode-swift: userdocs/userdocs.docc/Articles/Advanced/install-pre-release.md

# Installing Pre-Release Builds

> Live on the bleeding edge by switching to the pre-release version.

Reviewer: maybe we cut this sentence? Think next paragraph says it in a more polished way. At the very least maybe we can come up with something different that "bleeding edge"

Author: In DocC, the first sentence after the header is a "sub-heading" (or whatever you want to call it). Shows up as a blurb under the title rather than it's own paragraph. So, it's more of a flavour text.

Reviewer 2: Yeah was also wondering the same as Adam. Not sure if the term "live on" would be familiar as well. Maybe something like:

> Try out the latest updates by switching to the pre-release version

> The Swift extension provides pre-release builds that can be used to test out unreleased features or get bug fixes before an official release can be published. Pre-release build numbers are always one minor version ahead of the most recent release.

Reviewer 3: ```suggestion
The Swift extension provides pre-release builds that you can use to test out unreleased features or get bug fixes before the project publishes an official release. Pre-release build numbers are one minor version ahead of the most recent release.
```

The "Switch to Pre-Release Version" button in the VS Code extensions view is used to install the latest pre-release builds:

> ![](install-pre-release.png)

Reviewer 3: ```suggestion
![A snapshot of VS Code that has Extensions highlighted, showing the Swift extension. In the detail panel of the extension view, a red box highlights the button "Switch to Pre-Release Version".](install-pre-release.png)
```

Always good to include alt-text for images.

> A "Switch to Release Version" button will appear in its place that lets you go back to the official release.

Reviewer 3: ```suggestion
When you select to use a pre-release version, the button changes to "Switch to Release Version", which lets you go back to the official release.
```
